service/history/workflow/context.go

```diff
@@ -958,19 +952,13 @@ func (c *ContextImpl) maxHistorySizeExceeded(shardContext shard.Context) bool {
 
 	if historySize > historySizeLimitError && c.MutableState.IsWorkflowExecutionRunning() {
 		c.logger.Warn("history size exceeds error limit.",
-			tag.WorkflowNamespaceID(c.workflowKey.NamespaceID),
-			tag.WorkflowID(c.workflowKey.WorkflowID),
-			tag.WorkflowRunID(c.workflowKey.RunID),
 			tag.WorkflowHistorySize(historySize))
 
 		return true
 	}
 
 	if historySize > historySizeLimitWarn {
 		c.throttledLogger.Warn("history size exceeds warn limit.",
-			tag.WorkflowNamespaceID(c.MutableState.GetExecutionInfo().NamespaceId),
-			tag.WorkflowID(c.MutableState.GetExecutionInfo().WorkflowId),
-			tag.WorkflowRunID(c.MutableState.GetExecutionState().RunId),
```

Comment:
I see no reason why `c.MutableState.GetExecutionInfo()` is used here since the mutable state is loaded from the workflow key. So this should be equivalent to the workflow key.